linode/manager
# M3-4297 IP Table

## Description

This PR contains the combined IPv4-IPv6 table under the Linode Network Tab.

So far I've carried the actions over from what's in production now. A future PR will organize the action menus of the various IP types and add the action buttons.

Still to do:
- [x] Tighten up CSS (the padding and the border before the empty cell for the action menu)
- [X] Write a few tests for the new helper functions.

To test:

Try out accounts in various IP states:

- With multiple public IPv4s
- With/without private IPv4s
- With reserved IPv4s (initiate cross DC migration)
- With shared IPv4s
- With IPv6 disabled
- With IPv6 Ranges

I also made some CSS changes that affect the Linodes Landing table; so please check that as well.
